src/lemmy_federation_exporter/__main__.py

```diff
@@ -29,8 +29,13 @@
 
 # TODO: This should probably be split a bit to improve readability.
 # TOOD: Afterwards the noqa marker for PLR0915 can be removed.
-async def metrics(request: aiohttp.web.Request) -> aiohttp.web.Response:  # noqa: PLR0915
+async def metrics(
+    request: aiohttp.web.Request,
+) -> aiohttp.web.Response:  # noqa: PLR0915
+
+    # Get request query paramaters
     instance = request.query.getone("instance")
+    filter_instance = request.query.get("filter_instance", "none")
```

Comment:
```suggestion
    remote_instances_filter = request.query.get("remote_instances")
    if remote_instances_filter is not None:
        remote_instances_filter = remote_instances_filter.lower().split(",")
```

considering that it's probably not easy/possible to include multiple  `remote_instance` parameters in prometheus, i'd use a single `remote_instances` parameter as comma separated string, which, when set should replace the fediseer filter.

i don't think there's a use case for filtering both on a static list and also fediseer?